Jenkinsfiles/build.groovy

```diff
@@ -19,9 +19,10 @@
 //   -> hudson.model.DirectoryBrowserSupport.CSP : removes restrictions on CSS file load, thus html pages of test reports are displayed pretty
 //   See: https://docs.openshift.com/container-platform/3.9/using_images/other_images/jenkins.html for a complete list of JENKINS env vars
 // define constants
-def COMPONENT_NAME = 'bcrs-entities-create-ui'
+def COMPONENT_NAME = 'coops-ui'
```

Comment:
Can we rename this to 'filings-ui' or 'businesses-ui' (per future route (*))?

(*) depends what Create UI route will be

Reply:
I assume Create UI URL will be .../business/create .

It would be nice to name our components similarly. Thoughts?

Reply:
Or maybe name these components according to GH repo name?

Reply:
I would've liked that too, but whoever set it up originally when it was just coops chose coops-ui so its following that name convention everywhere now and it needs to be consistent across everything because of how its parameterized. If we altered this we'd need to recreate all the openshift stuff too

Reply:
OK, no change for now.